pom.xml

```diff
@@ -216,7 +216,7 @@
         <version.org.syslog4j>0.9.30</version.org.syslog4j>
         <version.org.wildfly.build-tools>1.1.0.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.4.Final</version.org.wildfly.checkstyle-config>
-        <version.org.wildfly.core>2.0.1.Final</version.org.wildfly.core>
+        <version.org.wildfly.core>2.0.2.Final-SNAPSHOT</version.org.wildfly.core>
```

Comment:
This won't work. :)

It doesn't seem to be needed though. I was able to revert this line and do a build, at least through the smoke tests.